### PR TITLE
support skipping between image files

### PR DESCRIPTION
**Pull request by adipose**

### CODE REVIEW

File: src/mpc-hc/MainFrm.cpp [hunk 1]
```diff
@@ -4894,6 +4888,40 @@ DROPEFFECT CMainFrame::OnDropAccept(COleDataObject* pDataObject, DWORD dwKeyStat
...
+    CStringW ext(path.GetExtension().MakeLower());
+    return IsImageFileExt(ext);
+}
+
+bool CMainFrame::IsImageFileExt(CStringW ext) {
+    ext.MakeLower();
+    return (ext == _T(".jpg") || ext == _T(".jpeg") || ext == _T(".png") || ext == _T(".gif") || ext == _T(".bmp") || ext == _T(".tiff") || ext == _T(".jpe") || ext == _T(".tga"));
+}
+
+bool CMainFrame::IsPlaylistFileExt(CStringW ext) {
```
Comment (clsid2) [inline src/mpc-hc/MainFrm.cpp:4905]: IsPlaylistFileExt name would make me think it checks for playlist file extension (.mpcpl .m3u etc)

Just IsPlayableFormatExt would be enough for now? Rest is redundant.

File: src/mpc-hc/MainFrm.cpp [hunk 2]
```diff
@@ -4894,6 +4888,40 @@ DROPEFFECT CMainFrame::OnDropAccept(COleDataObject* pDataObject, DWORD dwKeyStat
     return DROPEFFECT_NONE;
 }
 
+bool CMainFrame::IsImageFile(CStringW fn) {
+    CPath path(fn);
+    CStringW ext(path.GetExtension().MakeLower());
```
Comment (clsid2) [inline src/mpc-hc/MainFrm.cpp]: MakeLower() is redundant here

**Comment by clsid2** [comment]
These should also be added to image ext list:
.heic .avif

**Comment by adipose** [comment]
I think that fixes all issues.  Only extra call is `IsAudioOrVideoFileExt`.  Maybe useful for clarity.

One thought.  Should audio-only skip to audio-only only, and video skip to video only?  Or they can skip between audio/video in same folder?

**Comment by clsid2** [comment]
> One thought. Should audio-only skip to audio-only only, and video skip to video only? Or they can skip between audio/video in same folder?

Maybe. I am not sure. Some may like that some may not. Lets leave as is for now. Some extensions like .ogg can be both audio and video, which is also problematic.

**Pull request merged**